src/server.rs

```diff
@@ -50,7 +50,7 @@ impl Server {
 
         info!("Runtime server listening on {}", self.config.sock_path());
 
-        let rt = MyRuntime::default();
+        let rt = RuntimeServiceImpl::<PinnedSandbox>::default();
```

Comment:
@mrunalp the idea is to select the sandbox implementation already on runtime creation and always use "this kind of sandbox" for the whole server. This way we can already select the type on server creation.

Reply:
while the runtime performance may benefit, this may not work. For instance, even if we do use PinnedSandbox type, if a runtime class of kata shows up, we have to run with an infra container. I think we may need to be more dynamic

Reply:
That’s a very good point. So we have to change the runtime based on the runtime class from the request, right? I’ll change the implementation to be more flexible.

Reply:
We can define an enum for the 3 types of Pods. Based on the config/runtime class we pick the right implementation to pass create the Sandbox with.

Reply:
I removed the type constrained from `MyRuntime` and switched to a builder pattern for the sandbox.